Lib/selectors.py

```diff
@@ -457,6 +481,31 @@ def select(self, timeout=None):
                     ready.append((key, events & key.events))
             return ready
 
+        def modify(self, fileobj, events, data=None):
+            try:
+                key = self._fd_to_key[self._fileobj_lookup(fileobj)]
+            except KeyError:
+                raise KeyError("{!r} is not registered".format(fileobj)) from None
+            changed = False
+            if events != key.events:
+                selector_events = 0
+                if events & EVENT_READ:
+                    selector_events |= select.EPOLLIN
+                if events & EVENT_WRITE:
+                    selector_events |= select.EPOLLOUT
+                try:
+                    self._epoll.modify(key.fd, selector_events)
+                except BaseException:
+                    super().unregister(fileobj)
```

Comment:
The doc says that modify() works as unregister+register, so basically the doc is still ok after your change.

Question: is it worth it to mention that modify() unregisters the FD on error, and maybe even write an unit test for that?